java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/SparkPlanner.scala

```diff
@@ -317,7 +326,7 @@ class SparkPlanner(session: SparkSession, config: OpenmldbBatchConfig, dbName: S
     }
 
     try {
-      engine = new SqlEngine(sql, db, engineOptions)
+      engine = new SqlEngine(sql, dbs, engineOptions, config.defaultDb)
```

Comment:
Unit test for query on multiple databases

Reply:
Single unit tests has been added to test registering table with default db. But we can not use multiple databases now because we can not resolve the DataProviderNode with db name and table name.